src/Uno.Extensions.Reactive/Collections/Facades/Map/ConverterExtensions.cs

```diff
@@ -0,0 +1,119 @@
+	/// </summary>
+	public static void ArrayCopy<TFrom, TTo>(this IConverter<TFrom, TTo> converter, ICollection source, Array targetArray, int targetIndex)
+	{
+		var length = targetArray.Length;
+		var sourceArray = new TFrom[length];
+		source.CopyTo(sourceArray, 0);
+		for (var i = 0; i < length; i++)
+		{
+			targetArray.SetValue(converter.Convert(sourceArray[i]), i + targetIndex);
+		}
+	}
+
+	/// <summary>
+	/// Helper to implements the <see cref="ICollection.CopyTo"/> method using a <see cref="IConverter{TFrom, TTo}"/>.
+	/// </summary>
+	public static void ArrayCopy<TFrom, TTo>(this IConverter<TFrom, TTo> converter, ICollection source, TTo[] targetArray, int targetIndex)
+	{
+		var length = targetArray.Length;
+		var sourceArray = new TTo[length];
+		source.CopyTo(sourceArray, 0);
+		for (var i = 0; i < length; i++)
+		{
+			var item = sourceArray[i];
+			if (item is not null)
+			{
+				targetArray[i + targetIndex] = converter.Convert(item);
+			}
+		}
+	}
+
+	/// <summary>
+	/// Creates an equality comparer decorator over an <see cref="IConverter{TFrom,TTo}"/>. 
+	/// </summary>
+	/// <typeparam name="TFrom">Source instance type</typeparam>
+	/// <typeparam name="TTo">Target instance type</typeparam>
+	/// <param name="converter">The converter</param>
+	/// <param name="comparer">The decoratee</param>
+	/// <returns>An equality comparer which converts back the values using <paramref name="converter"/>, and then compare them using <paramref name="converter"/>.</returns>
+	public static IEqualityComparer<TTo> ToEqualityComparer<TFrom, TTo>(this IConverter<TFrom, TTo> converter, IEqualityComparer<TFrom> comparer)
+		=> new ConverterEqualityComparer<TFrom, TTo>(converter, comparer);
+
+	/// <summary>
+	/// Creates an equality comparer decorator over an <see cref="IConverter{TFrom,TTo}"/>. 
+	/// </summary>
+	/// <typeparam name="TFrom">Source instance type</typeparam>
+	/// <typeparam name="TTo">Target instance type</typeparam>
+	/// <param name="converter">The converter</param>
+	/// <param name="comparer">The decoratee</param>
+	/// <returns>An equality comparer which converts the values using <paramref name="converter"/>, and then compare them using <paramref name="converter"/>.</returns>
+	public static IEqualityComparer<TFrom> ToEqualityComparer<TFrom, TTo>(this IConverter<TFrom, TTo> converter, IEqualityComparer<TTo> comparer)
+		=> new ConverterEqualityComparer<TTo, TFrom>(converter.Inverse(), comparer);
+
+	/// <summary>
+	/// Creates a new converter which anonymize the convertion types
+	/// </summary>
+	/// <remarks>Null object won't be converted</remarks>
+	public static IConverter<object?, object?> ToConverter<TFrom, TTo>(this IConverter<TFrom, TTo> converter)
+		=> new ObjectConverter(
+			o => o is null ? default : converter.Convert((TFrom)o),
+			o => o is null ? default : converter.ConvertBack((TTo)o));
+
+	private class ObjectConverter : IConverter<object?, object?>
```

Comment:
record?

Reply:
I tried ... not real interest, so rollback and kept as `class` :)